.gitlab-ci.yml

```diff
@@ -37,7 +37,7 @@ stages:
     include:
       - local: '.gitlab/custom-jobs.yml'
       - project: 'radiuss/radiuss-shared-ci'
-        ref: v2022.06.0
+        ref: task/remove-intel-18
```

Comment:
Let’s merge `task/remove-intel-18`, and maybe tag the merge `v2022.06.1` before we merge things here.
@trws I know it seems like running down a rabbit hole, am I thankful for your hard work.

Reply:
Done